packages/ui-components/src/components/Icon/Icon.component.tsx

```diff
@@ -21,7 +21,7 @@ import ChevronLeft from "@material-design-icons/svg/outlined/chevron_left.svg"
 import ChevronRight from "@material-design-icons/svg/outlined/chevron_right.svg"
 import Close from "@material-design-icons/svg/filled/close.svg"
 import ContentCopy from "@material-design-icons/svg/outlined/content_copy.svg"
-import Danger from "./../Icon/icons/juno-danger.svg"
+import Danger from "../../deprecated_js/Icon/icons/juno-danger.svg"
```

Comment:
What is the reason that you load these icons from the from the `deprecated_js` folder here in the TypeScript Icon component? They are custom icons we created ourselves. We don't want to get rid of them.

Reply:
No, I just missed them out. Thank you for catching this. I have moved the svg folder to TS Icon component folder.